tests/libdnf/sack/AdvisoryTest.cpp

```diff
@@ -97,11 +97,9 @@ void AdvisoryTest::testGetApplicablePackagesModulesNotSetup()
 
     // When modules are not setup all advisory collections are applicable and we get all packages
```

Comment:
I just realized this comment is no longer true, and this test is identical to: `testGetApplicablePackagesMultipleApplicableCollections`, but it's not that important for the 8.5 branch.

I will propagate these fixes for the advisory unittests to `dnf-4-master`.